x/programs/rust/examples/counter/src/lib.rs

```diff
@@ -151,13 +131,6 @@ mod tests {
             params: vec![Param::String(PROGRAM_PATH.into())],
             require: None,
         });
-        plan.add_step(Step {
-            endpoint: Endpoint::Execute,
-            method: "initialize_address".into(),
-            max_units: 1000000,
-            params: vec![counter_id.into(), bob_key.clone()],
-            require: None,
-        });
 
         plan.add_step(Step {
             endpoint: Endpoint::Execute,
```

Comment:
Could you please also remove the call to `initialize_address` in the `external_call` test ? It's not ran for now but may save us 1.5 seconds once we fix it 😄

Reply:
I'll be the one re-enabling that test so I deal with it then